sdk-python/littlehorse/workflow.py

```diff
@@ -445,6 +501,21 @@ def searchable_on(
     def required(self) -> "WfRunVariable":
         self._required = True
         return self
+    
+    def with_default(self, default_value: Any) -> WfRunVariable:
+        self.__set_default(default_value)
+
+        return self
+
+    def __set_default(self, default_value: Any) -> None:
```

Comment:
1 underscore for internal 2 for language related ones.

example `__init__` 2 underscore